src/config.rs

```diff
@@ -108,33 +105,21 @@ impl TryFrom<&Path> for ImageConfig {
 
 #[derive(Clone, Debug, Deserialize)]
 pub struct Paths {
+    /// sigstore config file for simple signing
+    pub sigstore_config: String,
+
     /// Path to `Policy.json`
     pub policy_path: String,
```

Comment:
Here, `policy_path` should be `policy_uri`, right? So it should be fixed as `kbs://example.org/default/image_policy/latest` or some else.

So as `auth_file`.

Reply:
Yes. it is set to `"kbs:///default/security-policy/test"`, same as defined in AA